packages/web/hooks/use-export.ts

```
@@ -0,0 +1,590 @@
+        onProgress?.(75)
+        setState((prev) => ({ ...prev, progress: 75 }))
+
+        // Handle the result
+        if (autoDownload && result.success) {
+          exportService.downloadFile(result)
+        }
+
+        onProgress?.(100)
+
+        // Convert lib/export ExportResult to types/export ExportResult
+        let content = ''
```

Biome (2.1.2)

[error] Shouldn't redeclare 'content'. Consider to delete it or rename it.

'content' is defined here:


(lint/suspicious/noRedeclare)